### ISSUE TITLE
Able to access sites from non-encrypted local dns sevrer

### ISSUE THREAD

**Issue by RandomModderJDK** [quindecim/dnscrypt-proxy-android#8]
**Is your feature request related to a problem? Please describe.**
I am not able to use a free wifi, which changes the ip address of the captive portal page. I need to relog every day, so I have to disable the proxy, reboot, get the ip address, add it to the captive-portals.txt., reboot. 

**Describe the solution you'd like**
A way to still use force_tcp and use the forwarding rules. If there is a solution already in the config to specify a working fallback server, please tell me.

**Describe alternatives you've considered**
I tried to use the fallbacks_server feature, but it didn't work. It just tried to connect to the normal ones and gave up.

**Comment by yolobnb** [comment]
Does the IP address of the wifi change every day?

**Comment by RandomModderJDK** [comment]
Yeah, so I'd need to query the local dns at 10.0.0.1 for the captive portal
